### PR TITLE
How-to-guides AWS DocumentsDB docs update

### PR DESCRIPTION
**Pull request by garvit3835**
-  Created the steps required for new and existing DBs.
- Added three images in the document.

### CODE REVIEW

**Comment by mishushakov** [comment]
Are those ready for review?

**Comment by vikasgoswami2** [comment]
@mishushakov yes please start your review.

File: how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+### Step 1: Accessing DocumentDB from a DevBox
+
+1. Follow the [Connecting to AWS](../../existing-network/connecting-to-aws.md) guide.
+2. Go to **Amazon DocumentDB > Clusters > Your Cluster.**
+3. Select **Connectivity & Security.** You will see the commands and DocumentDB hostname for accessing it.
+
+![image](../../../.gitbook/assets/documentDB-connectivity-and-security.png)
+
+4. Go to **DevBox**.
+5. Import the MongoDB public GPG key:
```
Comment (mishushakov) [inline how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md:19]: Merge the two steps together:
In your DevBox, import the MongoDB public GPG key

File: how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+```
+7. Reload local package database:
+```sh
+sudo apt-get update
+```
+8. Install mongo shell:
+```sh
+sudo apt-get install -y mongodb-org
+```
+9. Download certificate:
```
Comment (mishushakov) [inline how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md:37]: the certificate

File: how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+![image](../../../.gitbook/assets/documentDB-connectivity-and-security.png)
+
+4. Go to **DevBox**.
+5. Import the MongoDB public GPG key:
+```sh
+curl -fsSL https://www.mongodb.org/static/pgp/server-7.0.asc | \
+   sudo gpg -o /usr/share/keyrings/mongodb-server-7.0.gpg \
+   --dearmor
+```
+6. Create a list file for MongoDB:
```
Comment (mishushakov) [inline how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md:25]: Add MongoDB package repository source

File: how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+```sh
+echo "deb [ arch=amd64,arm64 signed-by=/usr/share/keyrings/mongodb-server-7.0.gpg ] https://repo.mongodb.org/apt/ubuntu jammy/mongodb-org/7.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-7.0.list
+```
+7. Reload local package database:
+```sh
+sudo apt-get update
+```
+8. Install mongo shell:
```
Comment (mishushakov) [inline how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md:33]: MongoDB

File: how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+```sh
+sudo apt-get install -y mongodb-org
+```
+9. Download certificate:
+```sh
+wget https://truststore.pki.rds.amazonaws.com/global/global-bundle.pem
+```
+10. Connect to mongo:
```
Comment (mishushakov) [inline how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md:41]: to the database

File: how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+## New Database
+
+### Step 1: Creating a DocumentDB Instance
+
+1. Follow the [Connecting to AWS](../../existing-network/connecting-to-aws.md) guide.
+2. Go to **Amazon DocumentDB > Create Cluster**.
+3. Select **Instance Based Cluster**.
+4. In the **Configuration** section choose **Instance Class**, number of instances for your cluster.
+5. Set **Username and Password**
```
Comment (mishushakov) [inline how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md:58]: forgot a dot here

File: how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+6. Turn on **Show advanced settings**.
+7. In the **Network settings** , choose the VPC where your bastion host is running, subnet group and assign a security group which has inbound access to only VPC.
+8. Click **Create cluster**.
+
+![image](../../../.gitbook/assets/documentdb-cluster.png)
+
+### Step 2: Accessing DocumentDB from a DevBox
+
+1. Go to **DevBox**.
+2. Import the MongoDB public GPG key:
```
Comment (mishushakov) [inline how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md:68]: In your DevBox, import the MongoDB public GPG key

File: how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+```sh
+curl -fsSL https://www.mongodb.org/static/pgp/server-7.0.asc | \
+   sudo gpg -o /usr/share/keyrings/mongodb-server-7.0.gpg \
+   --dearmor
+```
+3. Create a list file for MongoDB:
```
Comment (mishushakov) [inline how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md:74]: Add MongoDB package repository source

File: how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+```sh
+echo "deb [ arch=amd64,arm64 signed-by=/usr/share/keyrings/mongodb-server-7.0.gpg ] https://repo.mongodb.org/apt/ubuntu jammy/mongodb-org/7.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-7.0.list
+```
+4. Reload local package database:
+```sh
+sudo apt-get update
+```
+5. Install mongo shell:
```
Comment (mishushakov) [inline how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md:82]: MongoDB

File: how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+```sh
+sudo apt-get install -y mongodb-org
+```
+6. Download certificate:
```
Comment (mishushakov) [inline how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md:86]: the certificate

File: how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+```sh
+wget https://truststore.pki.rds.amazonaws.com/global/global-bundle.pem
+```
+7. Connect to mongo:
```
Comment (mishushakov) [inline how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md:90]: to the database

Comment (vikasgoswami2) [reply how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md]: @mishushakov please check now. Updated them as per your suggestions.

**Comment by vikasgoswami2** [comment]
@mishushakov Though DocumentDB got merged in staging. Still I am not able to see this on the UI. Is there any issue with the pipeline ?

**Pull request merged**